lib/SIL/SILConstants.cpp

```diff
@@ -472,53 +487,30 @@ ArrayRef<SymbolicValue> SymbolicValue::getAggregateValue() const {
 namespace swift {
 
 /// This is the representation of a constant enum value with payload.
-struct alignas(SymbolicValue) EnumWithPayloadSymbolicValue final {
```

Comment:
what does `alignas(SymbolicValue)` do (is it just a performance optimization)?

I copied it from the definition of `AggregateSymbolicValue`, which also have a `SymbolicValue` typed payload.

Reply:
Is is necessary for the TrailingArray case when the trailing array element needs an alignment stronger than the declared class.  EnumWithPayloadSymbolicValue doesn't have a trailing array, so I simplified it a bit.